charts/gardener/controlplane/charts/application/templates/validatingwebhook-admission-controller.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: {{ include "webhookadmissionregistration" . }}
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: validate-namespace-deletion
+  name: gardener-admission-controller
```

Comment:
Is helm able to deal with the renaming and clean up the webhook with the old name? Or do we need to adapt our machinery to explicitly delete the old webhook?

Reply:
I tested it on a local deployment setup where i initiated a gardener cluster without the changes and after making the changes and executing `make gardener-up` again it deleted the old VWC and created a new one with the new name.

Reply:
I was not asking about the local setup. I am asking for the internal machinery that we have to deploy the gardener controlplane component. Will it be able to deal with this renaming or do we need to adapt it to delete the webhook explicitly after this change?

Reply:
I see that we deploy this chart with `helm template` + `kubectl apply` afterwards. So the old resource won't be automatically cleaned up and we have to explicitly clean it up.
@rfranzke do you prefer to take over the corresponding adaptation as the v1.59 release responsible or do you prefer @AleksandarSavchev to prepare the corresponding change?

Reply:
Thank you!